### ISSUE TITLE
[Linear Progress Indicator] Indicator colour is gone when screen/text is zoom 

### ISSUE THREAD

**Issue by ashishjohn1908** [material-components/material-components-android#3856]
**Description:** Indicator colour is invisible when we try to zoom using Accessibility

**Android API version:** API 33

**Device:** Emulator API 33

https://github.com/material-components/material-components-android/assets/16619763/a9a5d0dc-f2ac-48f6-8e90-03f86b168932

**Comment by pekingme** [comment]
Did the "indicator" segment become transparent with a non-zero progress or the progress is reset to 0? Btw, which release are you using?

**Comment by ashishjohn1908** [comment]
Yes. I have tried to add toast to display the progress. It was showing the correct value in toast but no progress was visible in progress bar. I am using the latest release available.
I was facing the issue when I used in one of the container in a scroll view. Its working fine outside the scroll view.
